index.html

```diff
@@ -151,8 +285,8 @@ <h4>Artists and creative types</h4>
                   <h4>Donate to Pursuance!</h4>
                   <p>The Pursuance Project is a nonprofit organization of dedicated volunteers - it needs your financial support. <a class="text-primary" href="donate.html">Every bit helps!</a></p>
                   <h4>Join our Mailing List!</h4>
-                  <p>Our monthly mails include updates on the platform's development and ways to get involved. <a class="text-primary" href="#mc_embed_signup" data-toggle="modal">Sign up here!</a></p>
-                  <button class="subscribe-bottom"><a href="#mc_embed_signup" data-toggle="modal" class="nav-link">Subscribe to our mailing list here!</a></button>
+                  <p>Our monthly mails include updates on the platform's development and ways to get involved.</p>
```

Comment:
Let's not say monthly, as it could be more or less frequent than that.

Reply:
Will change "Our monthly mails include" to "Our mailings include"

Also here's another q/a to be added to the faq if it's okay --

**Q**: When will the Pursuance Project be ready for the general public to use?
**A**: An ambitious list of features and components integration is planned for the foreseeable future starting with:
- Closed Alpha testing (through June/July 2018)
- Closed Beta testing (August through October 2018)
- **Public v1.0 launch** (late October / early November 2018)
- Continued integration and adoption of latest technologies, as they become available, for Pursuance v2.0 (mid 2019)
- Security audit of the web app and native apps, complete secure integration across all social media platforms, continued iteration over all aspects of the UX (2019 - 2021)

Note - that answer's paraphrased from the gist /pursuance_feature_roadmap.md (which has a github tag "secret") so is it okay to phrase it that way and/or include a "For feature/component details see [link-to-gist]" line after the bullet points?